relay-server/src/actors/project_upstream.rs

```diff
@@ -58,7 +64,7 @@ impl UpstreamQuery for GetProjectStates {
     }
 
     fn path(&self) -> Cow<'static, str> {
-        Cow::Borrowed("/api/0/relays/projectconfigs/?version=3")
+        Cow::Borrowed("/api/0/relays/projectconfigs/?version=4")
```

Comment:
**IMPORTANT**: Please leave this unchanged for now. Continue to request v3 for project configs and use v4 ONLY for global configs.